internal/config/validate_test.go

```diff
@@ -65,6 +65,10 @@ func TestConfigValidatePipelines(t *testing.T) {
 			"pipeline_multi_step.hcl",
 			"",
 		},
+		{
+			"pipeline_nested_pipes.hcl",
```

Comment:
Should the test files `pipeline_nested.hcl` and `pipeline_nested_refs.hcl` also be included?